applications/Profond.md

```diff
@@ -0,0 +1,154 @@
+- [https://github.com/bagasprakasa](https://github.com/bagasprakasa)
+- [https://github.com/internnos](https://github.com/internnos)
+- [https://github.com/adeyusupp](https://github.com/adeyusupp)
+
+### Team LinkedIn Profiles (if available)
+
+- [https://www.linkedin.com/in/adhigunamahendra/](https://www.linkedin.com/in/adhigunamahendra/)
+- [https://www.linkedin.com/in/bagasprakasa/](https://www.linkedin.com/in/bagasprakasa/)
+- [https://www.linkedin.com/in/irfianto/](https://www.linkedin.com/in/irfianto/)
+- [https://www.linkedin.com/in/amajidsinar/](https://www.linkedin.com/in/amajidsinar/)
+- [https://www.linkedin.com/in/ekkirinaldi/](https://www.linkedin.com/in/ekkirinaldi/)
+- [https://www.linkedin.com/in/vincentsalaka/](https://www.linkedin.com/in/vincentsalaka/)
+- [https://www.linkedin.com/in/adeyusupp/](https://www.linkedin.com/in/adeyusupp/)
+
+## Development Status :open_book:
+
+PoC → Here’s our current MVP in Sui [http://console.profond.ai/](http://console.profond.ai/)
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** Duration of the whole project (e.g. 2 months)
+- **Full-Time Equivalent (FTE):** 2 FTE
+- **Total Costs:** $10.000
+
+### Milestone 1 — No code smart contract builder
+
+- **Estimated duration:** 1 month
+- **FTE:** 2
+- **Costs:** 4,000 USD
+
+| Number | Deliverable | Specification |
+| --- | --- | --- |
+| 0a. | License | MIT |
+| 0b. | Documentation | We will provide documentation page on how to create NFT (PSP34) and FT (PSP22) smart contract with our interface. API documentation for project settings will also be provided.  |
+| 0c. | Testing and Testing Guide | All functions will be covered with full integration test, testing guide will also be provided. |
+| 0d. | Article | We will publish an article that explains the no code builder |
+| 1. | No Code NFT Creation (ink!) | Allows users to create NFT collection smart contract (PSP34) using ink! in one of WASM supported Parachains. |
+| 2. | No Code Fungible Token Creation (ink!) | Allows users to create Fungible Token smart contract (PSP22) using ink! in one of WASM supported Parachains. |
+| 3. | Project Management Module | Allows users to create more than one projects |
+| 4. | Multichain Integration | Allows users to choose between Polkadot Parachains, Kusama, or Sui |
+
+### Milestone 2 Example — Analytics and API
+
+- **Estimated Duration:** 1 month
+- **FTE:** 2
+- **Costs:** 6,000 USD
+
+| Number | Deliverable | Specification |
+| --- | --- | --- |
+| 0. | API Documentation | We will provide documentation page on how to create basic NFT collection (PSP34) using our no-code builder and creating a fungible token (PSP22). |
+| 1. | Analytic Functionality | Add on-chain analytic for the smart contract. |
+| 2.  | REST API Service | API for users to get the data analytic. |
+| 3. | Indexer Service | Create an Indexer service. |
+| 4. | Go To Market | Onboard 20 project within a month after release. |
```

Comment:
I'm not sure this is something that is covered by the grants program, you can probably remove the "go to market" part.

Reply:
Sure, removed this row.